api/src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+export const host = process.env.API_HOST || "master-api";
+export const port = process.env.PORT || 8080;
+export const isSsl = process.env.USE_SSL === 'ssl' ? true : false;
```

Comment:
@galethil ESLint complains about single-quote usage. It likes double quotes more.

Reply:
You are right. I fixed all linting issues.